spec/jobs/upvs/fetch_services_with_forms_list_job_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe Upvs::FetchServicesWithFormsListJob, type: :job do
+  describe '#perform' do
+    let(:url) { 'https://example.com' }
+
+    let(:downloader) { double }
+
+    it 'downloads and imports ServicesWithForms list in V1 format' do
+      expect(downloader).to receive(:download_file).with(url).and_return(fixture_filepath('upvs/services-v1.csv'))
+
+      subject.perform(url, downloader: downloader)
+
+      expect(Upvs::ServiceWithForm.first).to have_attributes(
+        instance_id: 2107,
+        external_code: 'App.GeneralAgenda',
+        meta_is_code: 'NULL',
+        name: 'Veobecná agenda',
+        type: 'Formulárové sluby',
```

Comment:
tuto mas (asi) zly charset, na nasej strane treba ukladat vsetko ako utf-8 (podobne nizsie/vyssie)

```suggestion
        type: 'Formulárové služby',
```

Reply:
To je chzba na ich strane, v `CSV`. Pozeram ze maju zly encoding

Reply:
@jsuchal co s tymto? report?

Reply:
@smith558 nemusia mat nutne zly encoding, predpokladam, ze to nie je utf-8, skusal si ci to je nieco ine?

inde to riesme takto https://github.com/slovensko-digital/datahub/blob/master/app/jobs/vszp/import_debtors_list_job.rb#L18

Reply:
Prave pozeram u nich ze fakt to maju zle asi. Ich CSV `ContentType` header nema ziadny `charset` attr takze vyskusam rozne encodings

![image](https://user-images.githubusercontent.com/44023416/68309880-087a6980-00a7-11ea-936d-7bbac81cc549.png)

Reply:
@smith558 Windows-12xx ani ISO-8859-x nic? 

ale kludne im napis, ze im tom chyba v Content-Type a nech ti povedia aky charset to je...

Reply:
@smith558 nejaky posun tu? toto je kriticke celkom

Reply:
Je to rozbite na ich strane. Ziadny encoding to nevyriesil. Chcem to nahlasit len uz dlhsie vidim toto.

![image](https://user-images.githubusercontent.com/44023416/68346601-75642280-00ec-11ea-82d9-e5762236ac79.png)

Idem poslat mail priamo.

Reply:
ok, cakame

Reply:
@smith558 @pavolzbell necakame, encoding reported, fixneme to potom. teraz nam na spustenie stacia tie fieldy co chybu nemaju